apps/agent/src/mastra/workflows/github-notify-developer-about-pr-status.ts

```diff
@@ -188,27 +189,38 @@ const notifyDeveloperAboutPRStatus = new Workflow({
         )
 
         return {
-          todayPRs: todayPRsWithReviews.map((pr) => ({
-            number: pr.number,
-            title: pr.title,
-            url: pr.html_url,
-            author: pr.user.login,
-            createdAt: pr.created_at,
-            updatedAt: pr.updated_at,
-            mergedAt: pr.merged_at,
-            isMerged: pr.merged_at !== null,
-            isWaitingForReview: githubClient.isWaitingForReview(pr),
-            hasMergeConflicts: githubClient.hasMergeConflicts(pr),
-            isApprovedWaitingForMerging:
-              githubClient.isApprovedButNotMerged(pr),
-            draft: pr.draft,
-            isWIP: githubClient.isWIP(pr),
-            labels: pr.labels.map((label) => label.name),
-            reviewers: pr.requested_reviewers.map((reviewer) => reviewer.login),
-            hasComments: pr.comments > 0 || pr.review_comments > 0,
-            hasReviews: pr.reviews && pr.reviews.length > 0,
-            body: pr.body,
-          })),
+          todayPRs: todayPRsWithReviews.map((pr) => {
+            const author =
+              `<@!${githubIdMapper.getDiscordID(pr.user.login)?.toString()}>` ||
+              pr.user.login
+            const reviewers = pr.requested_reviewers.map(
+              (reviewer) =>
+                `<@!${githubIdMapper.getDiscordID(reviewer.login)?.toString()}>` ||
+                reviewer,
```

Comment:
I believe it’s best to map the githubID when creating the message embed. This way, it can be used later, such as for making a GitHub author link.